### DISCUSSION TITLE
Pause at layer / M 600

### DISCUSSION THREAD

**Discussion by Switch-Witch** [OrcaSlicer/OrcaSlicer#477] [Q&A]
Hi, I am searching a good way to implement a pause control at a layer hight or the m600 control. The way I found out is to slice the object, to slide to the specific layer hight and to right click on the slider to open the sub menu. As often I try I was never able to get the right layer hight as the slider moves when I try to do a rightclick on it. Is there any other way to implement the controls at a specific layer hight?

**Comment by madscientist57** [comment]
I just did that, by doing the right click on the green bar below the position indicator.  I used the scroll wheel to get to the exact layer, then the right click.  I don't know what to do next, but I got that far.  I ran across your question while trying to find out.  Some guy on youtube said to then add a couple of lines in the gcode to define the length of the pause, but dammit Jim I'm an engineer not a coder.
